EncounterMe/EncounterMe/Views/IndividualObjectPage.xaml.cs

```diff
@@ -105,6 +105,7 @@ static string AllObjects(List<MapPin> list, MapPin currentPin, int i, Filter fil
             string Objects = "";
             Location loc = new Location() { Latitude = currentPin.Latitude, Longitude = currentPin.Longitude };
 
+            if(filter == IsClose)
```

Comment:
Labai keistai atrodo šitas be riestinių skliaustelių :D ir pas mus daugumoj vietų tarp 'if' ir sąlygos yra tarpelis, nes tokia vs rekomendacija, tai aš dažnai prieš keliant darbą dar paspaudžiu ctrl + k + d, kad automatiškai patvarkytų tokias vietas, jei jų yra

Reply:
Čia išvis šito neturėjo būt :D matyt kažką galvojau daryt parašiau ir pamiršau vėliau ištrint :D